cmd/docker-driver/README.md

```diff
@@ -3,7 +3,7 @@
 ## Overview
 
 Docker logging driver plugins extends Docker's logging capabilities. You can use Loki Docker logging driver plugin to send
-Docker container logs directly to your Loki instance or [Grafana Cloud](https://grafana.com/loki).
+Docker container logs directly to your Loki instance or [Grafana Cloud](https://grafana.com/oss/loki).
```

Comment:
https://grafana.com/loki already redirects to https://grafana.com/oss/loki and may redirect elsewhere in the future. I don't see a need to do this.